_data/team/roles/non_core_library_maintenance.yml

```diff
@@ -0,0 +1,80 @@
+role: Non-core library maintenance
+description: |
+  The non-core library maintenance team is responsible for the
+  managing the maintenance of the all MDAnalysis packages
+  that are not the core MDAnalysis library.
+tasks:
+  - Managing issues and pull-requests for sub-projects
+  - Managing new features and updates
+  - Managing maintenance
+  - Working with CI and CD team and infrastructure
+subroles:
+
+  - subrole: distopia
+    current_leads: []
+    current_members: []
+    historical_members:
+      - Hugo MacDermott-Opeskin
+    
+
+  - subrole: cookiecutter-mdakit
+    current_leads:
+      - Lily Wang
+    current_members: []
+    historical_members:
+      - Lily Wang
+    
+
+  - subrole: mda-encore
+    current_leads: []
+    current_members: []
+    historical_members:
+      - Ian Kenney
```

Comment:
I think mdaencore will transition out of this list, but let's start in this way.